doc/userguide/devguide/contributing/backports-guide.rst

```diff
@@ -0,0 +1,107 @@
+========================
+Suricata Backports Guide
+========================
+
+This document describes the processes used to backport content to previous
+Suricata releases. Most often, this means security and/or bug fixes;
+however, in some cases, features may be backported to previous Suricata releases.
+
+There are multiple versions of Suricata at any given time:
+    * Master
+    * Current release
+    * Release prior to current release
```

Comment:
Not sure about the words here. Perhaps something like 
* master
* major stable 1 ...